vueapp/src/stores/playerStore.ts

```diff
@@ -9,6 +9,7 @@ export const usePlayerStore = defineStore(PLAYERS_STORE_ID, {
   state: () => ({ allPlayers: [] as Player[], allMembers: [] as Member[]}),
   getters: {
     waitingPlayers: (state) => state.allPlayers, // TODO filter only players that are not already in a game
+    selectableMembersForWaitingList: (state) => state.allMembers.filter(member => !state.allPlayers.find(player => player.name === member.name))
```

Comment:
Just adding a note for future improvements
```suggestion
    selectableMembersForWaitingList: (state) => state.allMembers.filter(member => !state.allPlayers.find(player => player.name === member.name)) // TODO: match on id instead of name... move `id` property from `Member` to `Player` class and devise a way to assign a unique id to guests.  Members should already have unique ids
```